test/acceptance/use-cases/test-scripts/create-acl.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const app = require('../../../../');
+const expect = require('../../../helpers/expect');
+const fs = require('fs-extra');
+const clone = require('lodash').clone;
+const loopback = require('loopback');
+const path = require('path');
+const testSupport = require('../../../helpers/test-support');
+const util = require('util');
+const workspaceManager = require('../../../../component/workspace-manager');
+
+const ModelAccessControl = app.models.ModelAccessControl;
+
+app.on('booted', function() {
+  app.emit('ready');
+});
+
+module.exports = function() {
+  const testsuite = this;
+
+  this.When(/^I add acl to model '(.+)'$/,
```

Comment:
Given
  - an acl with these configs
    - with what role user (ROLE)
    - with permission set to ALLOW or DENY (the ACL)
    - operation execute on find
AND:
  - we have a user that is not logged (authenticated or unauthenticated)
When I execute the find operation on a model with